src/rhsm/config.py

```diff
@@ -85,7 +85,8 @@
     "splay": "1",
     "disable": "0",
     "auto_registration": "0",
-    "auto_registration_interval": "60",
+    "auto_registration_interval": "1",
+    "auto_registration_identity_interval": "60",
```

Comment:
Can we actually reduce this default to something like 10 minutes?  I don't know why it was 60 minutes before, but it is quite long.

Also, second question: why is this value set to 1 in `etc-conf/rhsm.conf`, and 60 here? should they not have the same value?